rmc-docs/src/install-guide.md

```diff
@@ -0,0 +1,85 @@
+# RMC Installation Guide
+
+RMC has not yet reached the point where releases are available, and so to use RMC you must currently build from source.
+
+In general, the following dependencies are required:
+
+1. The dependencies needed to built `rustc`. RMC is a fork of the Rust compiler, and so we have the same minimum requirements.
+2. CBMC (>= 5.30.1)
+3. CBMC Viewer (>= 2.6)
+
+## Installing on Ubuntu 20.04
+
+We recommend trying out RMC with Ubuntu 20.04.
+The simplest way to install is following our CI scripts:
+
+```
+git clone git@github.com:model-checking/rmc.git
+cd rmc
+git submodule update --init
```

Comment:
Is this command necessary (I don't recall running it in my initial setup). I just did that, and it's been running for over 30 minutes for me and seems to be cloning a lot of repositories:
```
Cloning into '/home/zyadh/rmc/firecracker'...
Cloning into '/home/zyadh/rmc/library/backtrace'...
Cloning into '/home/zyadh/rmc/library/stdarch'...
Cloning into '/home/zyadh/rmc/src/doc/book'...
Cloning into '/home/zyadh/rmc/src/doc/edition-guide'...
Cloning into '/home/zyadh/rmc/src/doc/embedded-book'...
Cloning into '/home/zyadh/rmc/src/doc/nomicon'...
Cloning into '/home/zyadh/rmc/src/doc/reference'...
Cloning into '/home/zyadh/rmc/src/doc/rust-by-example'...
Cloning into '/home/zyadh/rmc/src/doc/rustc-dev-guide'...
Cloning into '/home/zyadh/rmc/src/llvm-project'...
```

Reply:
I've assumed it was necessary, since our CI is doing it.